CHANGES.md

```diff
@@ -2858,7 +2859,7 @@
   - `RegionMapping`: Used instead of TableDataSource for region-mapped csvs.
   - `DataTable` and `DataVariable` have been replaced with new classes, `TableStructure` and `TableColumn`.
   - `RegionProvider`: `loadRegionsFromWfs`, `processRegionIds`, `applyReplacements`, `findRegionIndex` have been made internal functions.
-  - `RegionProviderList`: `chooseRegionProvider` has been changed and renamed `getRegionDetails `.
+  - `RegionProviderList`: `chooseRegionProvider` has been changed and renamed `getRegionDetails`.
```

Comment:
Deleting this space improves the changelog. Did you mean to delete the other space? Did you mean to delete this one 😆 ?

Reply:
These were done by Prettier! Ill keep an eye on it in future